src/Comps/style/header.css

```diff
@@ -0,0 +1,24 @@
+header {
```

Comment:
### Your Design
![student-navbar (copy)](https://user-images.githubusercontent.com/93039359/191902314-fd3fcdea-9916-4535-b1b2-28bd9e1432f0.png)

### Template Design
![bookstore-design (copy)](https://user-images.githubusercontent.com/93039359/191902271-84ae2185-240f-480d-9e22-735c285d64ba.png)


- [ ] As you can see in the design that the navbar contents are aligned with the books but in your design they are not. Kindly follow the design guidlines :+1:

Reply:
This change has been done well